vtu11/impl/parallel_helper_impl.hpp

```diff
@@ -0,0 +1,69 @@
+﻿//          __        ____ ____
+// ___  ___/  |_ __ _/_   /_   |
+// \  \/ /\   __\  |  \   ||   |
+//  \   /  |  | |  |  /   ||   |
+//   \_/   |__| |____/|___||___|
+//
+//  License: BSD License ; see LICENSE
+//
+#ifndef VTU11_PARALLEL_HELPER_IMPL_HPP
+#define VTU11_PARALLEL_HELPER_IMPL_HPP
+
+#include "vtu11_impl.hpp"
+namespace vtu11
+{
+  template<typename Writer>
+  void writePVTUfile( const std::string& path,
+                      std::string baseName,
```

Comment:
```suggestion
                      const std::string& baseName,
```
`std::string` should be passed by const-ref and not by value